content/kubermatic/main/references/reference_setup/_index.en.md

```diff
@@ -0,0 +1,478 @@
+
+- [MetalLB BGP Configuration](https://metallb.io/configuration/_advanced_bgp_configuration/)
+
+- [Cilium BGP Control Plane](https://docs.cilium.io/en/stable/network/bgp-control-plane/bgp-control-plane/) 
+
+
+## **Public/Private Cloud Load Balancers**
+
+For other Load Balancer scenarios we strongly recommend to use cloud environment specific Load Balancer what comes with the dedicated cloud CCM. This native cloud LBs can interact dynamically with Kubernetes to provide updates for service type Load Balancer or Ingress objects. For more detail see [Kubernetes - Services, Load Balancing, and Networking](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer).
+
+\
+
+
+# **DHCP / Networking**
+
+![](networking.png)
+
+- DHCP for worker nodes is required
+
+- Verify what Load Balancing Options your target environment has, see section “Load Balancing”
+
+- Direct Node-to-Node communication is needed on each cluster level (master/seed and user cluster level) based on generic Kubernetes requirements: [Kubernetes - Cluster Networking](https://kubernetes.io/docs/concepts/cluster-administration/networking/)
+
+- Check the more detailed requirements of the network at:
+
+  - [****KKP Documentation > Concepts > Networking****](https://docs.kubermatic.com/kubermatic/main/architecture/concept/kkp-concepts/networking) ****
+
+  - [****KKP Documentation > Networking > Control Plane Expose Strategy****](https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/networking/expose-strategies/)
+
+\
+
+
+# **DNS**
+
+Kubermatic requires DNS entries for NGINX and user clusters API servers. These have to be created after the Load Balancers, see [EE Installation - DNS Records](https://docs.kubermatic.com/kubermatic/main/installation/install-kkp-ce/#dns-records) and [EE Seed Cluster - DNS Records](https://docs.kubermatic.com/kubermatic/main/installation/install-kkp-ce/add-seed-cluster-ce/#dns-record).
+
+The NGINX server provides access to the Kubermatic UI and the logging and monitoring services (Prometheus, Kibana, etc.). It requires a DNS name like kubermatic.example.com and \*.kubermatic.example.com (e.g. for prometheus.kubermatic.example.com). Optional: if no wildcard DNS can be provided, dedicated DNS entries per exposed Service need to get entered (\~5 services).
+
+To access a user cluster via API, a wildcard DNS entry per seed cluster (in your case the master cluster is the only seed) has to be provided. E.g., \*.cluster-1.kubermatic.example.com. User clusters would be accessible via \[cluster-id].cluster-1.kubermatic.example.com. 
+
```

Comment:
check domain formating, better do it with  code  formating `dns-name`

Reply:
Done